src/main/java/io/github/jroy/happybot/levels/Leveling.java

```diff
@@ -63,7 +63,7 @@
   private LoadingCache<String, Integer> levelCache = CacheBuilder.newBuilder()
       .expireAfterAccess(10, TimeUnit.MINUTES)
       .maximumSize(200)
-      .build(new CacheLoader<>() {
+      .build(new CacheLoader<String, Integer>() {
```

Comment:
Why: since java 9 we can infer these just fine

Reply:
this was because i couldn't run it in java 8 but i'll remove it now